File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        reverse(s.begin(),s.end());
        transform(s.begin(),s.end(),s.begin(),::tolower);
        int n=s.size();
        int i=0;
        for(i=0;i<n;i++){
            if(s[i]>='a' && s[i]<='z') break;
        }
        int res=0;
        for(;i<n;i++){
            if(s[i]>='a' && s[i]<='z'){
                res++;
            }
            else break;
        }
        return res;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        res=0
        s=s[::-1].strip()
        for x in s:
            if(x==' '): break
            else: res+=1
        
        return res
        """
        :type s: str
        :rtype: int
        """
        